src/app/static/src/app/store/translations/locales.ts

```diff
@@ -347,6 +351,8 @@ const fr: Partial<Translations> = {
     projectName: "Nom du projet",
     projects: "Projets",
     remove: "supprimer",
+    renameProject: "Rename project",
```

Comment:
I'd normally put in our rough French translations here, I don't think we should start putting any English into the French section. My rudimentary French knowledge + Google suggests 'Renommer le projet' and 'Entrez un nouveau nom pour le projet' (+ 's'il vous plaît'? - not sure if the 'please' seems weird in French). I usually run whatever I come up with past Rich.